.packit.yaml

```diff
@@ -0,0 +1,23 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+upstream_project_url: https://github.com/namhyung/uftrace
+specfile_path: uftrace.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - uftrace.spec
+    - .packit.yaml
+
+# name in upstream package repository
+upstream_package_name: uftrace
+# downstream (Fedora) RPM package name
+downstream_package_name: uftrace
+
+jobs:
+- job: copr_build
+  trigger: pull_request
```

Comment:
It'd be nice if I can trigger it when I push some codes to a branch.  For example, I often push some code to the branches under `check/` or `review/`.  I usually don't create a PR by myself.

Also I want to run different set of tests depending on the trigger type.  For PR check, maybe it's enough run the tests on the latest Fedora only.  For nightly test, it can run on most versions.

Reply:
Yes, it's possible to trigger builds by commits to specific branches:
https://packit.dev/docs/configuration/#packit-service-jobs
Example snippet for `.packit.yml` from this page:
```yaml
- job: copr_build
  trigger: commit
  branch: main
  targets:
  - centos-stream-8-x86_64
  - centos-stream-9-x86_64
  - fedora-all
```
So yes, depending on the branch, you can configure specific targets to be built.

I don't know or understand yet, where the builds will be linked so you can access them, but I assume that it just has to be tried, and then we'll find out.

Reply:
Is it also possible to trigger the build manually like from nightly tests?

Reply:
There seems to be a possiblitily run jobs on a daily basis. At least, there is a project for daily LLVM snapshot builds:
https://copr.fedorainfracloud.org/coprs/g/fedora-llvm-team/llvm-snapshots/

A likewise also interesting topic is the Fedora testing farm which can be also used thruogh Packit:
https://fedoramagazine.org/test-github-projects-with-github-actions-and-testing-farm/

For public users, the tests are run only by AWS instances, but it has a nice test results viewer: 
https://docs.testing-farm.io/general/0.1/test-results.html

Reply:
Can these files live in the `misc/` directory?